utils/usePopper.ts

```diff
@@ -72,14 +87,13 @@ export function usePopper(options) {
           pointerEvents: 'none'
         }
       : {
-          position: state.data.offsets.popper.position,
           ...state.data.styles
         };
 
   return {
     style,
     placement: state.placement,
     outOfBoundaries: state.data && state.data.hide,
-    scheduleUpdate: popperInstance && popperInstance.scheduleUpdate
```

Comment:
Looks like TypeScript caught a 🐛here. 

The `popperInstance` is not an instance of `PopperJS`. Rather, it is a ref object that holds an instance of `PopperJS`. This expression always evaluated to and returned `undefined` — `scheduleUpdate` was never returned.

I updated the code to perform a lookup on the ref's current property because that is where the popper instance is located.